OptimizelySDK.Tests/ConfigTest/HttpProjectConfigManagerTest.cs

```diff
@@ -50,16 +50,21 @@ public void Setup()
         [Test]
         public void TestHttpConfigManagerRetreiveProjectConfigByURL()
         {
+            var t = MockSendAsync(TestData.Datafile);
             HttpProjectConfigManager httpManager = new HttpProjectConfigManager.Builder()
                 .WithUrl("https://cdn.optimizely.com/datafiles/QBw9gFM8oTn7ogY9ANCC1z.json")
                 .WithLogger(LoggerMock.Object)
                 .WithPollingInterval(TimeSpan.FromMilliseconds(1000))
                 .WithBlockingTimeoutPeriod(TimeSpan.FromMilliseconds(500))
                 .WithStartByDefault()
                 .Build();
+            t.Wait(1000);
```

Comment:
Do we need to wait here if we are mocking the request? Can we continue using `httpManager.OnReady()`? It seems like we only need to wait on something because the operation is async